nixops/backends/libvirtd.py

```diff
@@ -222,28 +243,33 @@ def start(self):
             self.private_ipv4 = self._parse_ip()
         else:
             self.log("starting...")
-            self._logged_exec(["virsh", "-c", "qemu:///system", "start", self.vm_id])
+            self.dom.create()
             self._wait_for_ip(0)
 
     def get_ssh_name(self):
-        assert self.private_ipv4
+        # assert self.private_ipv4
```

Comment:
Why is the assert commented out?

Reply:
I removed the assert as there are cases it fetches the ip on demand.